foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -30,8 +30,16 @@ Finally, this is your first JavaScript program built from scratch, so don't forg
    console.log(playRound(playerSelection, computerSelection));
    ~~~
 
-6. Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-   1. You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
+6. Write a NEW function called `game()`. Call the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
```

Comment:
Can we say playRound instead of previous function?

Reply:
Absolutely.